src/service-worker-registration.ts

```diff
@@ -115,36 +114,33 @@ function checkValidServiceWorker (swUrl: string, config?: Config) {
     headers: { 'Service-Worker': 'script' }
   })
     .then((response) => {
-    // Ensure service worker exists, and that we really are getting a JS file.
+      // Ensure service worker exists, and that we really are getting a JS file.
       const contentType = response.headers.get('content-type')
-      if (
-        response.status === 404 ||
-      (contentType != null && contentType.indexOf('javascript') === -1)
-      ) {
-      // No service worker found. Probably a different app. Reload the page.
+      if (response.status === 404 || (contentType != null && contentType.indexOf('javascript') === -1)) {
+        // No service worker found. Probably a different app. Reload the page.
         navigator.serviceWorker.ready.then((registration) => {
-          registration.unregister().then(() => {
-            window.location.reload()
-          }).catch((error: Error) => console.error(error))
+          registration.unregister()
+            .then(() => window.location.reload())
+            .catch((error: Error) => console.error(error))
         }).catch((error: Error) => console.error(error))
       } else {
-      // Service worker found. Proceed as normal.
+        // Service worker found. Proceed as normal.
         registerValidSW(swUrl, config)
       }
     })
-    .catch(() => {
-      console.log('No internet connection found. App is running in offline mode.')
-    })
+  .catch(() => {
+    console.log('No internet connection found. App is running in offline mode.')
```

Comment:
Do we want to store a field `offline` in the redux (for example in the new notifications-object)? This way we could later show a "You're in offline-mode. Live-changes won't be applied."-message.